README.md

```diff
@@ -1,56 +1,78 @@
+Using `pgzip` is the same as using the built-in `gzip` module.
 
-***In theoretical, compression and decompression acceleration should be linear according to the CPU cores. In fact, the performance is limited by IO and program language implementation.***
+Compressing data and writing it to a file:
 
-## Usage
-Use same method as gzip module
 ```python
 import pgzip
 
 s = "a big string..."
 
-## Use 8 threads to compress.
-## None or 0 means using all CPUs (default)
-## Compression block size is set to 200MB
+# An explanation of parameters:
+# `thread=8` - Use 8 threads to compress. `None` or `0` uses all cores (default)
+# `blocksize=2*10**8` - Use a compression block size of 200MB
 with pgzip.open("test.txt.gz", "wt", thread=8, blocksize=2*10**8) as fw:
     fw.write(s)
+```
+
+Decompressing data from a file:
+
+```python
+import pgzip
+
+s = "a big string..."
 
 with pgzip.open("test.txt.gz", "rt", thread=8) as fr:
     assert fr.read(len(s)) == s
 ```
 
 ## Performance
-### Compression:
+
+### Compression Performance
+
 ![Compression Performance](CompressionBenchmark.png)
 
-### Decompression:
+### Decompression Performance
+
 ![Decompression Performance](DecompressionBenchmark.png)
 
-*Brenchmarked on a 24 cores, 48 threads server (Xeon(R) CPU E5-2650 v4 @ 2.20GHz) with 8.0GB FASTQ text file.*
+Decompression was benchmarked using an 8.0GB `FASTQ` text file with 48 threads across 24 cores on a machine with Xeon(R) E5-2650 v4 @ 2.20GHz CPUs.
 
-*Using parameters thread=42 and blocksize=200000000*
+The compressed file used in this benchmark was created with a blocksize of 200MB.
 
 ## Warning
-**This package only replace the 'GzipFile' class and 'open', 'compress', 'decompress' functions of standard gzip module. It is not well tested for other class and function.**
 
-**As the first release version, some features are not yet supported, such as seek() and tell(). Any contribution or improvement is appreciated.**
+`pgzip` only replaces the following methods of `gzip`'s `GzipFile` class:
+
+- `open()`
+- `compress()`
+- `decompress()`
+
+Other class methods and functionality have not been well tested.
+
+Contributions or improvements is appreciated for methods such as:
+
+- `seek()`
+- `tell()`
 
 ## History
 
-This project is a fork of https://github.com/vinlyx/mgzip because we could not
-get in contact with vinlyx and we had bugfixes we wanted implemented. Thankyou
-to Vincent Li (@vinlyx) for all the hard work and if you ever come across this
-then feel free to get in contact.
+Created initially by Vincent Li (@vinlyx), this project is a fork of [https://github.com/vinlyx/mgzip](https://github.com/vinlyx/mgzip). We had several bug fixes to implement, but we could not him. The `pgzip` team would like to thank Vincent Li (@vinlyx) for his hard work. We hope that he will contact us when he discovers this project.
```

Comment:
Missing a word

but we could not **contact** him

I would also change all the 'he' and 'him' to 'they' and 'them'. You are probably correct to assume given the first name but you never know.

Reply:
I used Grammarly to check spelling and grammar and I'm quite surprised it missed that! Also, I don't contribute much on GitHub please pardon any silly mistakes I may make with pull requests, etc.

Reply:
No need to apologise.  This is the first package I have maintained publicly so I am learning as well.